NEWS.md

```diff
@@ -1,6 +1,7 @@
 # yardstick (development version)
 
 * Adds `sensitivity()` and `specificity()` as _aliases_ for `sens()` and `spec()` respectively, avoids conflict with other packages e.g. `readr::spec()`.
+* `roc_aunu()` and `roc_anup()` are two new ROC AUC metrics for multiclass classifiers. These measure the AUC of each class against the rest, `roc_aunu()` using the uniform class distribution (#69) and `roc_anup()` using the a priori class distribution (#70.).
```

Comment:
Small thing, I normally try and give a full blank line between bullet points. I think the rendered markdown looks a little cleaner

Reply:
Added in c509c4d6b93ba45cb0b4683408bdf3f7dc8d451c